docs/develop-resources/scenarios/new-field.md

```diff
@@ -11,8 +11,8 @@ REPO_ROOT="$(git rev-parse --show-toplevel)"
 cd $REPO_ROOT/dev/tools/controllerbuilder
 
 go run . update-types \
-    --parent-message "google.monitoring.dashboard.v1.Dashboard" \
-    --field-to-insert "row_layout" \
+    --parent "google.monitoring.dashboard.v1.Dashboard" \
+    --insert-field "row_layout" \
```

Comment:
I'm wondering if we should use an arg here rather than a flag. 
e.g. `go run . update-types  row_layout --parent <parent> --api-dir <apis/version/service> `

Reply:
Is using args the direction we are moving forward? So far I am just following the existing patterns.

In a WIP PR I am adding subcommands, which might make args look confusing. Example
```
# example usage of inserting a field
go run . update-types insert \
    --parent "google.monitoring.dashboard.v1.Dashboard" \
    --field "row_layout" \
    --api-dir ${REPO_ROOT}/apis/monitoring/v1beta1 \
    --ignored-fields "google.monitoring.dashboard.v1.PickTimeSeriesFilter.interval"

# example usage of syncing a message with all of its dependencies from proto package
go run . update-types sync \
    --service google.cloud.bigquery.datatransfer.v1 \
    --api-version bigquerydatatransfer.cnrm.cloud.google.com/v1beta1 \ 
    --message "google.cloud.bigquery.datatransfer.v1.TransferConfig"
```